test/controllers/slides_controller_test.rb

```diff
@@ -144,11 +144,19 @@ class SlidesControllerTest < ActionController::TestCase
     assert_equal "horizontal", slide.reload.orientation
   end
 
-  test "assert that `controls muted` attribute is present" do
-    video_foreground = slides.foreground_type
-    video_background = slides.background_type
+  test "assert that `controls muted` attribute is present on slides with foreground_type: video" do
+    # Arrange
+    super_admin = users(:super_admin)
+    slide = slides(:video_foreground)
+    assert_equal "video", slide.foreground_type
+
+    # Assume
 
-    assert_equal("video controls muted", slide.foreground_type)
-    assert_equal("video controls muted", slide.background_type)
+    # Act
+    get :preview, id: slide
+    assert_response :success
+
+    # Assert
+    assert_select "controls muted"
```

Comment:
This will not work. You're on the right track. But you need to research CSS selectors (these are the same things that jQuery use.)

Specifically, you'll want to figure out how to *select* a DOM element by tag *and* attribute. The tag in this case is the `<video />` tag. The attributes are `controls` and `muted`. It is a bit confusing in this case because the attributes have no value assigned to them. Here's a more common example of tag with attributes:

    <a href="/" valueless_attribute>label</a>

This is not much different than that.